channels/sy.m3u

```diff
@@ -1,21 +1,31 @@
 #EXTM3U x-tvg-url="http://195.154.221.171/epg/guidearab.xml.gz"
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/gXqzvLa.jpg" group-title="",ANN
 http://ns8.indexforce.com:1935/ann/ann/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/xBcBG2z.png" group-title="",Halab Today TV
+http://streaming.tootvs.com:1935/8010/8010/playlist.m3u8
 #EXTINF:-1 tvg-id="Orient TV ARB" tvg-name="Orient TV ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/qbgwnhB.png" group-title="",Orient TV
 https://svs.itworkscdn.net/orienttvlive/orienttv.smil/gmswf.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/ajsrKzA.png" group-title="",RTV Sat
-http://www.elahmad.com/tv/m3u8/syriatv.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/ajsrKzA.png" group-title="",RTV Sat
-http://www.elahmad.com/tv/m3u8/syriatv.m3u8?id=syria_news
```

Comment:
Could you explain why you deleted those links?

Reply:
Okay, but maybe we should leave these links as a backup?

Reply:
Done